.gitmodules

```diff
@@ -1,3 +1,8 @@
 [submodule "artwork"]
 	path = artwork
 	url = https://github.com/ReproNim/containers-artwork
+[submodule "sourcedata"]
+	path = sourcedata
+	url = https://github.com/ReproNim/ds000003-demo
+	datalad-id = 9040fae2-8704-11e8-aff3-0242ac12000c
+	datalad-url = https://github.com/ReproNim/ds000003-demo
```

Comment:
**WHAT FOR?**

Reply:
just a mistake from running the command here rather than in a tmp dir. Meant to squash this change out, will do next push